### ISSUE TITLE
CSV file incorrectly being matched as text/plain

### ISSUE THREAD

**Issue by melissafzhang** [gabriel-vasile/mimetype#36]
**Attach the file for which the detection is inaccurate**
You can use the file sharing site of your choice, just ensure the file will not expire or get deleted.

**Expected mime type**
text/csv

**Returned mime type**
text/plain
**Output of `go version`**
go1.12

**Additional context**
Here's the CSV file I'm trying to validate:
```
key,context,string
identity_verifications.legacy_complete.Success,,Success
identity_verifications.legacy_complete.Your information has been recorded. Thank you.,,Your information has been recorded. Thank you.
identity_verifications.legacy_complete.Return to Home Page,,Return to Home Page
identity_verifications.rate_limit.Try again later,,Try again later
identity_verifications.rate_limit.You can try again in %{time_from_now}.,,You can try again in %{time_from_now}.
"identity_verifications.rate_limit.We located your ID, but your question answers were wrong. You can try again in %{time_from_now}.",,"We located your ID, but your question answers were wrong. You can try again in %{time_from_now}."
"identity_verifications.rate_limit.We couldn't locate your ID, but you can try again in %{time_from_now}.",,"We couldn't locate your ID, but you can try again in %{time_from_now}."
"identity_verifications.rate_limit.We couldn't verify your identity right now, but you can try again in %{time_from_now}.",,"We couldn't verify your identity right now, but you can try again in %{time_from_now}."
shared.general.save,,Save
```

**Comment by gabriel-vasile** [comment]
I used the CSV in a [test](https://travis-ci.org/gabriel-vasile/mimetype/jobs/548167872) and it detected it as text/csv. Could it be that you are using an older release? CSV support was added in [v0.3.4](https://github.com/gabriel-vasile/mimetype/releases/tag/v0.3.4).

**Comment by gabriel-vasile** [comment]
Please ignore [the first build](https://travis-ci.org/gabriel-vasile/mimetype/builds/548162853). It failed because `go tool vet` was renamed to `go vet` in go1.12.

**Comment by melissafzhang** [comment]
Yup that was the issue. Thanks!

**Comment by melissafzhang** [comment]
My CSV files have comments at the top like this. When there are comments, the file is classified as text/plain.
```
# some comments
# hello
# see ya
key,context,string
identity_verifications.legacy_complete.Success,,Success
identity_verifications.legacy_complete.Your information has been recorded. Thank you.,,Your information has been recorded. Thank you.
identity_verifications.legacy_complete.Return to Home Page,,Return to Home Page
identity_verifications.rate_limit.Try again later,,Try again later
```

**Comment by gabriel-vasile** [comment]
Hello, comments in CSVs are now handled. The code is released in [v0.3.13](https://github.com/gabriel-vasile/mimetype/releases/tag/v0.3.13).
